BeeSwift/Settings/HealthKitConfigViewController.swift

```diff
@@ -3,14 +3,16 @@
 //  BeeSwift
 //
 //  Created by Andy Brett on 3/14/17.
-//  Copyright 2017 APB. All rights reserved.
+//  Copyright © 2017 APB. All rights reserved.
```

Comment:
👀 
Some MRs ago these were all removed

Reply:
Doh. I wonder how this merge conflict snuck in

Reply:
[swiftformat](https://github.com/nicklockwood/SwiftFormat?tab=readme-ov-file#file-headers) can be configured to adjust file headers and it could be run as part of a general format-code or ci-check/build step.